services/ui-src/src/utils/types/types.ts

```diff
@@ -1,13 +1,5 @@
 export type { IconType } from "react-icons";
 
-export enum CoreSetAbbr {
```

Comment:
haha we have like 3 or 4 PRs floating around right now between you, me, @gmrabian that remove a bunch of this stuff. 👍